### PR TITLE
Fix escaping of extensions, add extra filters for standard PHP functions

### PR DESCRIPTION
**Pull request by tobias-93**
Some small fixes to allow generation to be used with standard Symfony Twig environment

### CODE REVIEW

File: src/Extension/ExtraFilterExtension.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+   */
+  public function getFilters()
+  {
+    $options = ['is_safe' => ['html']];
+    return array(
+        'addslashes' => new TwigFilter('addslashes', 'addslashes', $options),
+        'var_export' => new TwigFilter('var_export', 'var_export', $options),
+        'is_numeric' => new TwigFilter('is_numeric', 'is_numeric', $options),
+        'ucfirst'    => new TwigFilter('ucfirst', 'ucfirst', $options),
+        'substr'     => new TwigFilter('substr', 'substr', $options),
```
Comment (bobvandevijver) [inline src/Extension/ExtraFilterExtension.php:23]: https://twig.symfony.com/doc/2.x/filters/slice.html

File: src/Extension/ExtraFilterExtension.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+   * {@inheritdoc}
+   */
+  public function getFilters()
+  {
+    $options = ['is_safe' => ['html']];
+    return array(
+        'addslashes' => new TwigFilter('addslashes', 'addslashes', $options),
+        'var_export' => new TwigFilter('var_export', 'var_export', $options),
+        'is_numeric' => new TwigFilter('is_numeric', 'is_numeric', $options),
+        'ucfirst'    => new TwigFilter('ucfirst', 'ucfirst', $options),
```
Comment (bobvandevijver) [inline src/Extension/ExtraFilterExtension.php:22]: https://twig.symfony.com/doc/2.x/filters/capitalize.html

File: src/Extension/ExtraFilterExtension.php [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+{
+  /**
+   * {@inheritdoc}
+   */
+  public function getFilters()
+  {
+    $options = ['is_safe' => ['html']];
+    return array(
+        'addslashes' => new TwigFilter('addslashes', 'addslashes', $options),
+        'var_export' => new TwigFilter('var_export', 'var_export', $options),
```
Comment (bobvandevijver) [inline src/Extension/ExtraFilterExtension.php:20]: https://twig.symfony.com/doc/2.x/functions/dump.html

File: src/Extension/ExtraFilterExtension.php [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+class ExtraFilterExtension extends AbstractExtension
+{
+  /**
+   * {@inheritdoc}
+   */
+  public function getFilters()
+  {
+    $options = ['is_safe' => ['html']];
+    return array(
+        'addslashes' => new TwigFilter('addslashes', 'addslashes', $options),
```
Comment (bobvandevijver) [inline src/Extension/ExtraFilterExtension.php:19]: Maybe https://twig.symfony.com/doc/2.x/filters/escape.html?

**Review by bobvandevijver — COMMENTED** [review]
I'm not sure about the extra filters...

**Comment by tobias-93** [comment]
I have just moved the `twigFilters` defined in the base builder to a dedicated extension class (so they can be used with all Twig environments, adding an extension works better than adding separate filters), did not want to change the syntax to prevent BC breaks.

**Comment by bobvandevijver** [comment]
Ok 👍🏻

**Pull request merged**